angelssmelting/prototypes/technology/smelting-cobalt.lua

```diff
@@ -89,7 +91,7 @@ data:extend(
       icon_size = 256, icon_mipmaps = 4,
       upgrade = true,
       prerequisites = {
-        --"angels-metallurgy-3",
+        "angels-metallurgy-4",
```

Comment:
`"angels-cobalt-casting-2"` correctly depends on `"angels-metallurgy-4"`.
However there is also a tech called `"angels-cobalt-smelting-2"` which is on the same tier as `"angels-cobalt-casting-2"`, but this tech does not yet depend on `"angels-metallurgy-4"`:
https://github.com/Arch666Angel/mods/blob/10f14c3176010cda7d979869bc6366c8c7d77f42/angelssmelting/prototypes/technology/smelting-cobalt.lua#L46-L53

Those prerequisites have to change to: a higher tier:
```lua
      prerequisites = {
        --"angels-metallurgy-4",
        "ore-processing-3",
```

Reply:
resolved in https://github.com/Arch666Angel/mods/pull/608/commits/7248ef7f0ea9cec61cc8e24961eb0fec40660863